src/api/routes/offer.js

```diff
@@ -28,6 +29,25 @@ module.exports = (app) => {
         }
     });
 
+    /**
+     * Gets an offer from the database with its id
+    */
+    router.get("/:offer", validators.get, async (req, res, next) => {
```

Comment:
rename to `:offerId` since that is actually what you will receive

Reply:
Note that this means renaming the usages of the parameter (the accesses to the `req` variable)